RELEASE.txt

```diff
@@ -35,7 +35,7 @@ CHANGED/ENHANCED EXISTING FUNCTIONALITY
   one from PEP 308 introduced in Python 2.5 (2006). The idiom being
   replaced (using and/or) is regarded as error prone.
 
-- MSVS: The default Windows powershell 7 path is added before the default
+- MSVC: The default Windows powershell 7 path is added before the default
```

Comment:
Needs a blurb about the user impact.  Perhaps something like.

"In some environments (such as Github Actions) this can yield a significant reduction in the time to initialize MSVC. (Without this change you may see > 10x slowdown) "

replace ##x slowdown with your best approximation of your test results.

The idea is that anyone reading this looking for a version with a fix should know from this blurb that it is indeed fixed.

RELEASE.txt is what gets sent to the mailing lists and other notification channels.  Users then would have to go to the CHANGES.txt.

So RELEASE.txt should have more or less the same content as this releases CHANGE.txt blurb, but organized differently, without the attribution.